src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/LegalRepOrganisationFormatter.java

```diff
@@ -63,6 +63,9 @@ public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCa
         if (organisationEntityResponse != null
             && StringUtils.isNotBlank(organisationEntityResponse.getOrganisationIdentifier())
             && featureToggler.getValue("share-case-feature", false)) {
+
+            log.info("PRD endpoint called for caseId [{}] ", callback.getCaseDetails().getId());
```

Comment:
here we can add OrgId we know it is not blank at this point

Reply:
Done.